ovsInit.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+#Start OVS in the Contiv container
+
+set -euo pipefail
+
+if [[ $(lsmod | cut -d" " -f1 | grep -q openvswitch) -eq 1 ]]; then
+	modprobe openvswitch
+fi
+
+mkdir -p /var/run/openvswitch
+mkdir -p /var/log/contiv
+
+sleep 2
```

Comment:
`mkdir` is atomic (except on NFS and other non-local storage) so why are we sleeping here?  Or is this for `modprobe`?  (if so, move up to `modprobe openvswitch && sleep 2`)